app/components/conversations/conversation-part-closed.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed, get } from '@ember/object';
+
+export default Component.extend({
+  classNames: ['conversation-part-closed'],
+
+  closedAt: null,
+
+  currentUser: service(),
+
+  isSelf: computed('author', 'user', function() {
+    return get(this, 'author.id') === get(this, 'user.id');
+  })
```

Comment:
Two problems here

## It's not clear what `author` is

You specify the value of author through the `.hbs`, as exampled in the test module you wrote:

```Handlebars
{{conversations/conversation-part-closed author=author closedAt=closedAt}}
```
When we do that, we also add that property into the `.js` and give it a default value, usually `null`, unless some other value makes sense. You already did this with `closedAt: null`, so you should do the same with `author: null`.

The code will work regardless, but we do this so that, when someone is reading the `.js`, they do not need to navigate back and forth between the `.js` and the `.hbs` and the component is relatively clear from the `.hbs` alone.

## `user` is not defined at all

The `author` is whichever user is responsible for this specific conversation part being created.

The `user` is supposed to be whichever user is currently signed in to the app. You get it from the `currentUser` service as `currentUser.user`, so to have it available to you as just `user`, you need to define an alias. You can look at the basic `conversation-part-comment` component to see how that's done.